packages/calendar-utils/package.json

```diff
@@ -40,11 +40,13 @@
   "devDependencies": {
     "moment": "2.29.4",
     "react": "17.0.2",
+    "react-dom": "17.0.2",
     "react-intl": "^6.3.2"
   },
   "peerDependencies": {
     "moment": "2.x",
     "react": "17.x",
+    "react-dom": "17.x",
```

Comment:
As we are "changing" the peer dependencies, hopefully changeset won't assume that this is a breaking change.

Reply:
Thanks for the heads-up, I had no idea about this. I'm not sure, but seems it's been fixed https://github.com/changesets/changesets/issues/380